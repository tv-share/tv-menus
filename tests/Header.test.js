import React from 'react';
import HeaderExample from "../examples/HeaderExample";
import renderer from 'react-test-renderer';

const fn = () => {};

it('renders correctely w/ logo click', () => {
    const component = renderer.create(
        <HeaderExample logoClickAction={fn}/>
    );
    
    expect(component.toJSON()).toMatchSnapshot();
});

it('renders correctely without logo click', () => {
    const component = renderer.create(
        <HeaderExample />
    );
    
    expect(component.toJSON()).toMatchSnapshot();
});