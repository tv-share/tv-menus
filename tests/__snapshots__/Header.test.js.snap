// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctely w/ logo click 1`] = `
<div
  className="header "
>
  <img
    className="logo pointer"
    onClick={[Function]}
    src="https://vignette.wikia.nocookie.net/logopedia/images/2/29/Windows_95_Logo.svg/revision/latest?cb=20160811163821"
  />
  <div
    className="title"
  >
    Apple
  </div>
  <ul
    className="items"
  >
    <li
      id="id 1"
      onClick={[Function]}
    >
      Item 1
    </li>
    <li
      id="id 2"
      onClick={[Function]}
    >
      Item 2
    </li>
  </ul>
  <ul
    className="options"
  >
    <li
      id="id 9"
      onClick={[Function]}
    >
      Option 2
    </li>
    <li
      id="id 10"
      onClick={[Function]}
    >
      <img
        src="https://t7.rbxcdn.com/d614151fd257cab3a05e4dc2f60524d4"
      />
    </li>
  </ul>
</div>
`;

exports[`renders correctely without logo click 1`] = `
<div
  className="header "
>
  <img
    className="logo "
    onClick={undefined}
    src="https://vignette.wikia.nocookie.net/logopedia/images/2/29/Windows_95_Logo.svg/revision/latest?cb=20160811163821"
  />
  <div
    className="title"
  >
    Apple
  </div>
  <ul
    className="items"
  >
    <li
      id="id 1"
      onClick={[Function]}
    >
      Item 1
    </li>
    <li
      id="id 2"
      onClick={[Function]}
    >
      Item 2
    </li>
  </ul>
  <ul
    className="options"
  >
    <li
      id="id 9"
      onClick={[Function]}
    >
      Option 2
    </li>
    <li
      id="id 10"
      onClick={[Function]}
    >
      <img
        src="https://t7.rbxcdn.com/d614151fd257cab3a05e4dc2f60524d4"
      />
    </li>
  </ul>
</div>
`;
